packages/stable/src/__tests__/api/unit/raw.unit.spec.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright 2020 Cognite AS
 
 import nock from 'nock';
-import CogniteClient from '../../cogniteClient';
```

Comment:
Any reason some unit tests file have been moved? could we move them back to stay consistent?

Reply:
I will them back